cfgov/unprocessed/apps/mmt-my-money-calendar/js/stores/strategies-store.js

```diff
@@ -172,14 +172,6 @@ class StrategiesStore {
     );
 
     if (results.length) return results;
```

Comment:
```suggestion
return results;
```

The code that uses `fixItResults` checks the `.length` of the computed property, it expects an array. Not having any return statement in this function would default to returning `undefined`.  So always returning the `results` array would be the right move here.
 
https://github.com/raft-tech/cfgov-refresh/blob/my-money-calendar/cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/strategies/fix-it/index.js#L109

Reply:
I see.  I will make the change.